### ISSUE TITLE
dates_times.am_pm.lowercase_periods is triggered when using uppercase letters

### ISSUE THREAD

**Issue by NateEag** [amperser/proselint#777]
Witness the following bash session:

```bash
$  proselint <(echo 'It is 6:00 PM, you know.')
/dev/fd/63:1:9: dates_times.am_pm.lowercase_periods With lowercase letters, the periods are standard.
$ proselint --version
0.8.0
```

Since the check is explicitly for lowercase letters, it should not be triggered when writing with uppercase letters.

I tried locally hacking [this call to `existence_check`](https://github.com/amperser/proselint/blob/dfaed0ca844a807388c863cdcbb3f4a384112a0e/proselint/checks/dates_times/am_pm.py#L23) to include an `ignore_case=False` keyword arg, but it does not seem to have changed the behavior.

**Comment by KVGarg** [comment]
Hi @NateEag , when you passed `ignore_case=False` as argument, you might not have changed the text due to which you got the same error again. This happened because the decorator `@memoize` memorize's the input text in cache due to which you get the same error again on passing same input text to the check. Try with different text by passing `ignore_case=False` as argument to `existence_check`.

I'm opening a PR with this correction in code.

**Comment by Nytelife26** [comment]
This will be fixed by the next release.
